_docs/01-racerd.md

```diff
@@ -16,7 +16,7 @@ RacerD statically analyzes Java code to detect potential concurrency bugs. This
 
 ## Triggering the analysis
 
-RacerD doesn't try to check *all* code for concurrency issues; it only looks at code that it believes can run in a concurrent context. There are two signals that RacerD looks for: (1) Explicitly annotating a class/method with `@ThreadSafe` and (2) using a lock via the `synchronized` keyword. In both cases, RacerD will look for concurrency issuess in the code containing the signal and all of its dependencies. In particular, it will report races between any non-`private` methods of the same class that can peform conflicting accesses. Annotating a class/interface with `@ThreadSafe` also triggers checking for all of the subclasses of the class/implementations of the interface.
+RacerD doesn't try to check *all* code for concurrency issues; it only looks at code that it believes can run in a concurrent context. There are two signals that RacerD looks for: (1) Explicitly annotating a class/method with `@ThreadSafe` and (2) using a lock via the `synchronized` keyword. In both cases, RacerD will look for concurrency issues in the code containing the signal and all of its dependencies. In particular, it will report races between any non-`private` methods of the same class that can peform conflicting accesses. Annotating a class/interface with `@ThreadSafe` also triggers checking for all of the subclasses of the class/implementations of the interface.
```

Comment:
Thanks for the PR. I don't see which part of the this paragraph was fixed.

Reply:
The first is a simple typo. Someone had added an extra s to issues to end up with issuess. The second is another typo,  where an n had been added to the word codebases so it currently says condebases. Let me know how else I can help. Happy to do what I can.

Reply:
Thanks, I just could not spot what was changed. I will import the changes.